docs/source/conf.py

```diff
@@ -1,105 +1,48 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# symbiflow-arch-defs documentation build configuration file, created by
-# sphinx-quickstart on Thu Oct 18 14:29:14 2018.
-#
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
+# Updated documentation of the configuration options is available at
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
 # import os
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
-# -- General configuration ------------------------------------------------
+# -- General configuration -------------------------------------------------------------------------
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
+project = 'symbiflow-arch-defs'
+copyright = '2018, Various'
+author = 'Various'
+
+# The short X.Y version.
```

Comment:
Can you add a TODO to pull these from `git-describe`?

Reply:
Done in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/2397/commits/0c0843195de232cd157b8bfb52bab5e2120e207b.